apps/client/src/components/atoms/text/Text.component.tsx

```diff
@@ -66,24 +73,49 @@ export function Text({
     className,
     children,
   }: SubTextProps): React.ReactElement => {
+    const content = (
+      <Linkify
+        componentDecorator={(
+          decoratedHref,
+          decoratedText,
+          key,
+        ): ReactElement => (
+          <a
+            target='_blank'
+            rel='noopener noreferrer'
+            href={decoratedHref}
+            key={key}
+            className='text-neutral-40 underline hover:no-underline'>
+            {decoratedText}
+          </a>
+        )}>
+        {children}
+      </Linkify>
+    );
+
     if (size === 'Medium') {
       return (
         <p style={style} className={`${className} font-medium text-indent`}>
-          {children}
+          {content}
         </p>
       );
     } else {
       return (
         <p style={style} className={className}>
-          {children}
+          {content}
         </p>
       );
     }
   };
 
   /** Related text size to the text variant */
   const textSizeValue = TextStyles.TEXT_MAP.get(textVariant);
-  const classNames = (colorClass || 'text-neutral-90') + ' ' + className;
+  const classNames =
```

Comment:
Why aren't you using `classnames` lib to build the classnames here?

Reply:
already fixed in 9953c3957419625c23d0ca0cccf993ac3ea6eea9 under PR #51